.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+ignore = E501
```

Comment:
Personally i'd add 
1. 251 (no spaces around parameter equals) - why does this get a special rule? 
2. 261 (2+ spaces before inline comment) - is there a reason 1 is unacceptable? 
3. 262, 265 Space after comment character. - I never knew this was a thing and it's jarring to me. That said, apparently this is an informal standard in a lot of languages and i'm the crazy one (according to reddit/SO posts on the topic). I probably won't follow it but i won't block/ignore any PRs that fix it, lol. 

Enforcing 901 would be ideal but i doubt it's practical.